ui/apps/platform/src/Containers/NetworkGraph/utils/simulatorUtils.ts

```diff
@@ -0,0 +1,34 @@
+import { NetworkPolicyModification } from 'Containers/Network/networkTypes';
+
+export function getDisplayYAMLFromNetworkPolicyModification(
+    modification: NetworkPolicyModification | null
+): string {
+    if (!modification) {
+        return '';
+    }
+    const { applyYaml, toDelete } = modification || {};
+    const shouldDelete = toDelete && toDelete.length > 0;
+    const showApplyYaml = applyYaml && applyYaml.length >= 2;
+
+    // Format toDelete portion of YAML.
+    let toDeleteSection;
+    if (shouldDelete && toDelete) {
```

Comment:
nit: `shouldDelete` is already false if `toDelete` is falsy, based on its initialization in line 10 above.
```suggestion
    if (shouldDelete) {
```

Reply:
Yea that's true. I overlooked that when I copy pasted it from somewhere else. Nice catch